src/Razor/src/Microsoft.VisualStudio.RazorExtension/UnifiedSettings/razor.registration.json

```diff
@@ -0,0 +1,121 @@
+{
+  "properties": {
+    "textEditor.razor.advanced.formatOnType": {
+      "type": "boolean",
+      "default": true,
+      "title": "@Setting_FormattingOnTypeDisplayName;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "description": "@Setting_FormattingOnTypeDescription;{13b72f58-279e-49e0-a56d-296be02f0805}",
+      "migration": {
+        "pass":{
+          "input": {
+            "store": "VsUserSettingsRegistry",
+            "path": "Razor\\FormatOnType"
```

Comment:
Does this json file support comments? I'm guessing this maps to `OptionsStorage.Collection` and `OptionsStorage.<setting name>`? A comment here to indicate that, or there to remind me to update this file, would be a good idea.

Reply:
Yea it does. I can add here, to say where the values come from